qiskit_aer/backends/aer_simulator.py

```diff
@@ -56,9 +56,9 @@ class AerSimulator(AerBackend):
         backend = AerSimulator(method='density_matrix',
                                 noise_model=noise_model)
 
-    **Simulating an IBMQ Backend**
+    **Simulating an IBM QUANTUM Backend**
 
-    The simulator can be automatically configured to mimic an IBMQ backend using
+    The simulator can be automatically configured to mimic an IBM QUANTUM backend using
```

Comment:
I think that we need to use `IBM Quantum` instead of `IBM QUANTUM`.

Reply:
Thanks for pointing that out. I have changed `IBM QUANTUM` to `IBM Quantum` in the latest commit. Can you verify it?